### DISCUSSION TITLE
人体系统调优不完全指南

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#51] [5.文章资料]
今日听闻左耳朵耗子（陈皓）因突发心肌梗塞不幸离世。感到非常惋惜。
于是分享这篇 zijie0 写的《[人体系统调优不完全指南](https://github.com/zijie0/HumanSystemOptimization)》给朋友们。调整自己形成健康的作息习惯，保持身心健康很重要。

**Comment by mistkafka** [comment]
惋惜！
